bookwyrm/models/user.py

```diff
@@ -356,8 +356,14 @@ def save(self, *args, **kwargs):
 
             # make users editors by default
             try:
-                self.groups.add(Group.objects.get(name="editor"))
-            except Group.DoesNotExist:
+                group = (
+                    apps.get_model("bookwyrm.SiteSettings")
+                    .objects.get()
+                    .default_user_auth_group
+                )
+                if group:
+                    self.groups.add(group)
+            except ObjectDoesNotExist:
```

Comment:
I think this try/except block won't be necessary now, since the value of `default_user_auth_group` can be none?

Reply:
Just tested this and the `ObjectDoesNotExist` exception is thrown in the tests given that the site settings are not (always?) initialised.

Reply:
ah that makes sense! I overlooked the `objects.get()` in there.